### DISCUSSION TITLE
Manual/Example-CreatingaBillboardPlane

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#563] [Notes]
# Manual/Example-CreatingaBillboardPlane

https://docs.unity3d.com/Manual/Example-CreatingaBillboardPlane.html

**Comment by vertxxyz** [comment]
Note that both the Material and the Mesh created in the example should be [Destroyed](https://docs.unity3d.com/ScriptReference/Object.Destroy.html) as they are native objects and will otherwise only be cleaned up when [`Resources.UnloadUnusedAssets`](https://docs.unity3d.com/ScriptReference/Resources.UnloadUnusedAssets.html) runs.
Also, a common mistake when using the Mesh API that is worth mentioning: the properties `vertices`, `triangles`, `normals`, and so on, all return **copies** of their underlying data. So do not directly call these properties in a for loop, as every iteration will allocate an entire new array.
